001-quarkus-getting-started/src/test/java/io/quarkus/qe/core/defaultprofile/MultipartResourceTest.java

```diff
@@ -0,0 +1,87 @@
+package io.quarkus.qe.core.defaultprofile;
```

Comment:
I don't like the split into controlprofile and defaultprofile.

It all can be in just `io.quarkus.qe.core` package.

control wording in `%control.quarkus.resteasy.multipart.input-part.default-charset=us-ascii` and MultipartResourceControlTestProfile etc. doesn't sound right.

AsciiMultipartResourceTestProfile, %ascii.quarkus.resteasy... sound more appropriate to the thing this profile is used to test.

Reply:
The name of the `control` profile was based on control sets in experiments, but I admit that in this case, it sounds odd. :).
The suggested name and wodring is better.

Reply:
The placement in separate packages was supposed to avoid switching back and forth between profiles during the test run (as described in the infobox here: https://quarkus.io/guides/getting-started-testing#testing_different_profiles).

Reply:
That's general suggestion, but in our case the is no reduction of Quarkus (re-)boots as `AsciiMultipartResourceTest` runs as first.

Reply:
I see, the rename conveniently changes the order of execution. Neat.